browser/brave_ads/tooltips/ads_tooltips_controller.h

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BRAVE_ADS_TOOLTIPS_ADS_TOOLTIPS_CONTROLLER_H_
+#define BRAVE_BROWSER_BRAVE_ADS_TOOLTIPS_ADS_TOOLTIPS_CONTROLLER_H_
+
+#include <map>
+#include <string>
+
+#include "brave/browser/profiles/profile_util.h"
+#include "brave/browser/ui/brave_tooltips/brave_tooltip.h"
+#include "brave/components/brave_adaptive_captcha/buildflags/buildflags.h"
+#include "brave/components/brave_ads/browser/ads_tooltips_delegate.h"
+
+namespace brave_adaptive_captcha {
```

Comment:
this should also be in a buildflag, but with the callback change this can probably go away

Reply:
Removed.